src/content/docs/react-components/codes/accordion.txt

```diff
@@ -0,0 +1,55 @@
+// Accordion.jsx
+import React, { useState } from 'react';
+import { motion, AnimatePresence } from 'framer-motion';
+
```

Comment:
Instead of dumping all text files inside the **CODE** folder. Please create folders.

Therefore for example: 
For Accordion => You have 2 files (accordion.txt & accordionApp.txt)
So create One more folder named **"Accordion"** under the **Code** folder. 

Inside this folder please have the txt files for specific components.

refer path like this:

`src/content/docs/react-components/codes/accordion/accordion.txt`
`src/content/docs/react-components/codes/accordion/accordionApp.txt`

Reply:
Please follow this for all other components